pom.xml

```diff
@@ -190,6 +203,7 @@
                     <target>${java-version}</target>
                     <testSource>${java-version}</testSource>
                     <testTarget>${java-version}</testTarget>
+                    <useIncrementalCompilation>false</useIncrementalCompilation>
```

Comment:
Without this some maven goals fail and it makes it harder to run the test (which is why this is a part of this PR).
I observed no performance loss, and  [apache](https://cwiki.apache.org/confluence/display/MAVEN/Incremental+Builds) and [maven users](https://stackoverflow.com/questions/8918165/does-maven-support-incremental-builds) seem to be on my side here.